contents/BenefitAdvertisingCards.js

```diff
@@ -143,21 +141,57 @@ const APPLICATION_CARD_CPP_DEATH_BENEFIT = {
   estimateLink: '',
 }
 
-function getAdvertsingCards() {
-  return [
-    APPLICATION_CARD_EI,
-    APPLICATION_CARD_CPP,
-    APPLICATION_CARD_OAS,
-    APPLICATION_CARD_GIS,
-    APPLICATION_CARD_CPPD,
-    APPLICATION_CARD_CPP_CHILDS_BENEFIT_AGED_18_25,
-    APPLICATION_CARD_CPP_SURVIVOR_PENSION_AND_CHILD_BENEFITS,
-    APPLICATION_CARD_CPP_ALLOWANCE_OR_ALLOWANCE_FOR_SURVIVOR,
-    APPLICATION_CARD_CPP_PENSION_SHARING,
-    APPLICATION_CARD_CPP_CREADIT_SPLIT,
-    APPLICATION_CARD_CPP_CHILD_REARING_PROVISION,
-    APPLICATION_CARD_CPP_DEATH_BENEFIT,
-  ]
+function getAdvertisingCards() {
```

Comment:
We'd want this function to simulate a result from AEM, and the prop `displayFlag` wouldn't be included in that.